src/openrct2-ui/input/MouseInput.cpp

```diff
@@ -110,7 +110,7 @@ void game_handle_input()
     invalidate_all_windows_after_input();
 
     int32_t state;
-    ScreenCoordsXY screenCoords;
+    ScreenCoordsXY screenCoords{};
```

Comment:
I know you did it with the best of intentions, but this doesn't relate to the task at hand, so can you please revert?
```suggestion
    ScreenCoordsXY screenCoords;
```